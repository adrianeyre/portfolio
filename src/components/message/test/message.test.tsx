import React from 'react';
import { shallow } from 'enzyme';

import IMessageProps from '../interface/message-props';
import Message from '../message';

describe('Message', () => {
	const defaultProps: IMessageProps = {
		message: 'My Message',
		type: 'type',
	};

	it('Should render correctly', () => {
		const login = shallow(<Message { ...defaultProps }/>);

		expect(login).toMatchSnapshot();
	});
});