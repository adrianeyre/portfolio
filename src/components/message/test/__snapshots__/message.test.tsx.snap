// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message Should render correctly 1`] = `ShallowWrapper {}`;
